src/native/corehost/test/CMakeLists.txt

```
@@ -2,7 +2,9 @@ add_subdirectory(fx_ver)
 add_subdirectory(mockcoreclr)
 add_subdirectory(mockhostfxr)
 add_subdirectory(mockhostpolicy)
-add_subdirectory(nativehost)
+if (NOT CLR_CMAKE_TARGET_ARCH_LOONGARCH64)
+    add_subdirectory(nativehost)
+endif()
```

Revert once https://github.com/dotnet/arcade/pull/15376 is merged and new image is published.
```suggestion
add_subdirectory(nativehost)
```